Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'bzip2-ruby', :github => 'chewi/bzip2-ruby', :branch => 'master'
```

Comment:
imho yes, but I am following the concept that closely. So, this has been just a poke/suggestion.